packages/desktop-client/src/components/accounts/Filters.js

```diff
@@ -335,16 +349,20 @@ export function FilterButton({ onApply }) {
       }
     }
 
-    let { error } = await send('rule-validate', {
-      conditions: [cond],
-      actions: [],
-    });
+    let { error } =
+      cond.field !== 'saved' &&
```

Comment:
It doesn't error out. It's written this way because the saved option sends an array of conditions where the rest of the options send only one condition. Without this line an error is triggered when a saved filter is chosen.